packages/renderer/src/lib/ui/TerminalWindow.svelte

```diff
@@ -14,7 +14,7 @@ const dispatch = createEventDispatcher<{ init }>();
 
 async function refreshTerminal() {
   // missing element, return
-  if (!logsXtermDiv) {
+  if (!logsXtermDiv || !window.getConfigurationValue) {
```

Comment:
it can't be the case, just need to mock up that in the tests

Reply:
I had tried that last time and failed, made better progress and then found Jeff had merged code yesterday I could copy. Fixed in new commit. :)